monai/transforms/transform.py

```diff
@@ -136,8 +137,15 @@ def apply_transform(
         Union[List[ReturnType], ReturnType]: The return type of `transform` or a list thereof.
     """
     try:
-        if isinstance(data, (list, tuple)) and map_items:
-            return [_apply_transform(transform, item, unpack_items, lazy, overrides, log_stats) for item in data]
+        if isinstance(data, (list, tuple)) and (map_items or type(map_items) is int):
+            # if map_items is an int, apply the transform to each item in the list `map_items` times
+            if type(map_items) is int and map_items > 0:
+                return [
+                    apply_transform(transform, item, map_items - 2, unpack_items, log_stats, lazy, overrides)
+                    for item in data
+                ]
+            else:
+                return [_apply_transform(transform, item, unpack_items, lazy, overrides, log_stats) for item in data]
```

Comment:
```suggestion
        map_items_ = int(map_items) if isinstance(map_items, bool) else map_items
        if isinstance(data, (list, tuple)) and map_items_ > 0:
            return [
                apply_transform(transform, item, map_items - 1, unpack_items, log_stats, lazy, overrides)
                for item in data
            ]
```

Simplified logic so that bool is considered a user friendly version of 1.